src/gluonts/model/deepar/_estimator.py

```diff
@@ -116,7 +117,9 @@ def __init__(
         self,
         freq: str,
         prediction_length: int,
-        trainer: Trainer = Trainer(),
+        trainer: Trainer = Trainer(
+            hybridize=True, num_batches_per_epoch=1, epochs=1
```

Comment:
These are not supposed to be in the PR I guess

Reply:
Ooops.